pkg/gardenlet/controller/controllerinstallation/controllerinstallation/reconciler.go

```diff
@@ -606,10 +606,7 @@ func (r *Reconciler) MutateSpecForSelfHostedShootExtensions(obj runtime.Object)
 			kubernetesutils.InjectKubernetesServiceHostEnv(podSpec.Containers, "localhost")
 		}
 
-		podSpec.Tolerations = append(podSpec.Tolerations,
-			corev1.Toleration{Operator: corev1.TolerationOpExists, Effect: corev1.TaintEffectNoSchedule},
-			corev1.Toleration{Operator: corev1.TolerationOpExists, Effect: corev1.TaintEffectNoExecute},
-		)
+		podSpec.Tolerations = append(podSpec.Tolerations, corev1.Toleration{Operator: corev1.TolerationOpExists, Effect: corev1.TaintEffectNoSchedule})
```

Comment:
Should we explicitly ignore the control plane role taint? 
With this toleration, we allow scheduling extensions to worker pools with explicitly specified taints.
I suggest using the same tolerations as for etcd-druid, mcm, and similar components

Reply:
True, makes sense. I think this is a left-over from back-and-forth with the bootstrap mode. Will do